source/gui/settingsDialogs.py

```diff
@@ -62,7 +63,6 @@
 	Set,
 	cast,
 )
-from url_normalize import url_normalize
```

Comment:
can we remove this from requirements then?

Reply:
No, as it's still used in `_SetURLDialog._normalize`
https://github.com/nvaccess/nvda/blob/bac98a373ee5addaedad391e656cbc0a51bff026/source/gui/_SetURLDialog.py#L214-L220